tools/dependencies/known-dependencies.txt

```diff
@@ -18,12 +19,16 @@ aggs-matrix-stats-client-7.5.1.jar
 aircompressor-0.10.jar
 airline-0.7.jar
 annotations-13.0.jar
+ant-1.6.5.jar
```

Comment:
Do we need add LICENSE in seatunnel-dist?

Reply:
yup, added.